lib/helpers/get-app-module-version.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var fs = require('fs');
+var path = require('path');
+
+var cachedModulePaths = {};
+var appModulesPath = path.join(path.dirname(fs.realpathSync(__filename)), 'node_modules');
```

Comment:
This still just gets the modules for this lib, and not the application. In my case, it's currently looking in: 

`/Users/timothy/Projects/stormpath/my-react-app/node_modules/express-stormpath/lib/helpers/node_modules/express`

But it should be looking in:

`/Users/timothy/Projects/stormpath/my-react-app/node_modules/express`

By changing this line to the following, I made it work:

```javascript
var appModulesPath = path.join(path.dirname(fs.realpathSync(require.main.filename)), 'node_modules');
```

But I don't know how safe it is to use `require.main.filename`. You can read more about it here: [Determine project root from a running node.js application](http://stackoverflow.com/questions/10265798/determine-project-root-from-a-running-node-js-application)